management-gateway/src/main/java/org/frankframework/management/gateway/HttpInboundGateway.java

```diff
@@ -174,8 +175,10 @@ public void init(WebSecurity builder) {
 	}
 
 	@Override
-	public void configure(WebSecurity builder) {
-		builder.addSecurityFilterChainBuilder(this::createSecurityFilterChain);
+	public void configure(WebSecurity webSecurity) {
+		SecurityFilterChain chain = createSecurityFilterChain(); //Create
+		SpringUtils.registerSingleton(applicationContext, "HttpInboundGateway-SecurityFilter", chain); //Register
```

Comment:
Is het niet simpeler / overzichtelijker om deze stap te doen in de method `createSecurityFilterChain()`, die toch al de applicationContext gebruikt?

Reply:
Daar heb ik mee geëxperimenteerd maar die wordt te laat aangeroepen (als lamba).